src/main/java/de/tum/in/www1/artemis/config/FileUploadExerciseProperties.java

```diff
@@ -0,0 +1,19 @@
+package de.tum.in.www1.artemis.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@ConfigurationProperties(prefix = "info.file-upload-exercise")
+@Configuration("fileUploadExerciseProperties")
+public class FileUploadExerciseProperties {
+
+    private String[] filePatterns;
```

Comment:
```suggestion
    private List<String> filePatterns;
```
Spring should be able to parse this as a list. It's more flexible that way imo and you don't have to call `Arrays.asList(...)` later on